internal/datacoord/garbage_collector_test.go

```diff
@@ -248,6 +249,14 @@ func Test_garbageCollector_scan(t *testing.T) {
 // initialize unit test sso env
 func initUtOSSEnv(bucket, root string, n int) (mcm *storage.MinioChunkManager, inserts []string, stats []string, delta []string, other []string, err error) {
 	paramtable.Init()
+
+	if Params.MinioCfg.UseSSL.GetAsBool() && len(Params.MinioCfg.SslCACert.GetValue()) > 0 {
+		err := os.Setenv("SSL_CERT_FILE", Params.MinioCfg.SslCACert.GetValue())
```

Comment:
Why is this env needed? cannot find any reference or usage of it.

Reply:
minio-go use this env to pass the certificate file path:
https://github.com/minio/minio-go/blob/e8ddcf0238962d766f44242b595511f4decd365c/transport.go#L73C22-L73C35